src/withStyles.jsx

```diff
@@ -58,12 +60,9 @@ export function withStyles(
 
         const addedProps = {
```

Comment:
Want to inline this into the JSX now? Sorry I didn't notice this earlier.

After this, I'll merge!